File: src/main/webapp/web/js/service/citiesService.js
//服务层
app.service('citiesService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../cities/findAll.do');		
	}	
	
	//读取列表数据绑定到表单中
	this.findCities=function(provinceId){
		return $http.get('../cities/findCities.do?provinceid='+provinceId);		
	}	
	
});

File: src/main/java/com/pinyougou/user/services/CitiesService.java
package com.pinyougou.user.services;
import java.util.List;
import com.pinyougou.pojo.TbCities;

import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface CitiesService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbCities> findAll();


	public List<TbCities> findByCities(String provinceid);
}
